mw-webapp/src/pages/reportsTable/ReportsTable.tsx

```diff
@@ -0,0 +1,45 @@
+import {useMemo} from "react";
+import {getCoreRowModel, HeaderGroup, RowModel, useReactTable} from "@tanstack/react-table";
+import {columns} from "src/component/table/columns";
+import {Table} from "src/component/table/Table";
+import {DayReport} from "src/model/businessModel/DayReport";
+import {useGetDataTable} from "src/service/TableService";
+
+/**
+ * Generated table data of {@link DayReport[]}
+ */
+export interface TableReports {
+  /**
+ * Group headers table
+ */
+  headerGroup: HeaderGroup<DayReport>[];
+  /**
+ * Object rows table
```

Comment:
not clear is it object or is it rows or is it table, the same, think how to clarify

Reply:
Done